cmd/peer-rest-server.go

```
@@ -905,6 +906,23 @@ func (s *peerRESTServer) ReloadPoolMetaHandler(mss *grid.MSS) (np grid.NoPayload
 	return
 }
 
+func (s *peerRESTServer) HandlerCleanupUploadIDCache(mss *grid.MSS) (np grid.NoPayload, nerr *grid.RemoteErr) {
+	objAPI := newObjectLayerFn()
+	if objAPI == nil {
+		return np, grid.NewRemoteErr(errServerNotInitialized)
+	}
+
+	pools, ok := objAPI.(*erasureServerPools)
+	if !ok {
+		return np, grid.NewRemoteErr(errors.New("not a pooled setup"))
+	}
+
+	uploadID := mss.Get(peerRESTCleanUploadID)
```

can we return an error if peerRESTCleanUploID is empty ? (easier for debugging issues)

---

Ack. Will add the same